### PR TITLE
Add Bulgarian Translation

### PR DESCRIPTION
**Pull request by haripetrov**

### CODE REVIEW

File: app/src/main/res/values-bg/strings.xml [hunk 1]
```diff
@@ -0,0 +1,746 @@
...
+    <string name="url_book" translatable="false">"http://nutritionfacts.org/book"</string>
+    <string name="donate">"Направете дарение"</string>
+    <string name="url_donate" translatable="false">"https://nutritionfacts.org/donate"</string>
+    <string name="subscribe">"Абонирай се"</string>
+    <string name="url_subscribe" translatable="false">"http://nutritionfacts.org/subscribe"</string>
+    <string name="open_source">"Отворен код"</string>
+    <string name="url_open_source" translatable="false">"http://nutritionfacts.org/open-source"</string>
+    <string name="nutritionfacts_title" translatable="false">"nutritionfacts.org"</string>
+    <string name="url_nutritionfacts" translatable="false">"http://nutritionfacts.org"</string>
+    <string name="name_john_slavick" translatable="false">"Джон Славик"</string>
```
Comment (slavick) [inline app/src/main/res/values-bg/strings.xml]: Please revert my name back to English. It is interesting to see it translated into Bulgarian though.

File: app/src/main/res/values-bg/strings.xml [hunk 2]
```diff
@@ -0,0 +1,746 @@
...
+    <string name="remind_me_at">"Напомни ми на"</string>
+    <string name="vibrate">"Вибрация"</string>
+    <string name="play_sound">"Възпроизвеждане на звук"</string>
+    <string name="videos">"Видео клипове"</string>
+
+    <!--
+    NOTE: These strings are used to created the url to take the user to the overall food videos page. These strings
+    must not be translated.
+    -->
+    <string name="food_info_videos_beans" translatable="false">"бобови растения"</string>
```
Comment (slavick) [inline app/src/main/res/values-bg/strings.xml]: This and the rest of the food_info_videos_* strings cannot be translated from English. Please revert all of these.

File: app/src/main/res/values-bg/strings.xml [hunk 3]
```diff
@@ -0,0 +1,746 @@
...
+    <string name="food_info_videos_spices" translatable="false">"подправки"</string>
+    <string name="food_info_videos_whole_grains" translatable="false">"зърнени"</string>
+    <string name="food_info_videos_beverages" translatable="false">"напитки"</string>
+    <string name="food_info_videos_exercise" translatable="false">"упражнения"</string>
+
+    <!--
+    NOTE: These strings should never be translated from English
+    These strings are used as the canonical names for the different foods and make it easy to switch between languages
+    -->
+    <string-array name="food_id_names" translatable="false">
```
Comment (slavick) [inline app/src/main/res/values-bg/strings.xml]: None of these strings should be translated. Please revert them.

File: app/src/main/res/values-bg/strings.xml [hunk 4]
```diff
@@ -0,0 +1,746 @@
...
+    </string-array>
+
+    <string-array name="food_info_serving_sizes_exercise">
+        <item>"90 минути умерена интензивност на физическа дейност"</item>
+        <item>"40 минути енергично интензивност дейност"</item>
+    </string-array>
+
+    <string name="error_cannot_handle_url">"Връзката не може да се отвори. Моля, инсталирайте браузър."</string>
+
+    <string-array name="food_videos_beans" translatable="false">
```
Comment (slavick) [inline app/src/main/res/values-bg/strings.xml]: Please revert all of these food_videos_* string-arrays back to English. Anything marked with translatable="false" should not be translated and should be left in English.

**Comment by haripetrov** [comment]
Done. I used Google Translator Toolkit and that is why I didn't see these strings: translatable="false"
Thanks, John for this feedback. As far as I can see now is OK to be merged. If you find something else just tell me.

**Comment by slavick** [comment]
I found a couple more issues when checking your updated pull request:
1. All translated strings are now surrounded by quotation marks. Please remove those.
2. In the "food_names" string-array, the last 5 foods are all set to @string/nuts

Once you fix those, I will check the pull request again. It should be good at that point.

Thanks

**Comment by haripetrov** [comment]
Fixed. If you find something else I will change it accordingly.

**Comment by marc-medley** [comment]
@haripetrov The translation process has now evolved such that an online application is available for anyone interested to be part of the _Daily Dozen App Translation Volunteer Team_. The team works with Steven Litrov (@salitrov) the Global Volunteer Director.

You, or others that you know, are welcome to apply:

* [https://nutritionfacts.org/volunteer/app-translation-volunteer/](https://nutritionfacts.org/volunteer/app-translation-volunteer/)

The current translations for both the Android and Apple Daily Dozen app can be viewed in the `daily-dozen-localization` repository in a `.tsv` spreadsheet form:

* [nutritionfactsorg/daily-dozen-localization repository](https://github.com/nutritionfactsorg/daily-dozen-localization)

Thank you for your patience with the Daily Dozen App Translation process evolution.

--Marc

**Pull request merged**